ngo_purchase_requisition/view/purchase_requisition.xml

```diff
@@ -114,18 +115,22 @@
                     <field name="name" string="Reference"/>
                     <field name="date_order" string="Order Date"/>
                     <field name="partner_id"/>
-                    <field name="company_id" groups="base.group_multi_company" widget="selection"/>
-                    <field name="location_id" groups="stock.group_locations"/>
                     <field name="minimum_planned_date" invisible="1"/>
-                    <field name="origin"/>
                     <field name="state"/>
                     <!-- purchase_requisition_bid_selection removed button purchase_cancel -->
                     <!-- purchase_requisition_bid_selection removed button purchase_confirm -->
                     <!-- purchase_requisition_bid_selection removed button purchase_approve -->
                     <!-- purchase_requisition_bid_selection change button icon -->
+                    <field name="bid_selection_remark"/>
+                    <button name="%(purchase_requisition_bid_selection.action_update_bid_selection_remark)d" type="action" string="Update selection remark" icon="gtk-ok"/>
```

Comment:
needs fixing: this is being changed in https://github.com/OCA/purchase-workflow/pull/111/files